react/week1/ToDo.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import ToDo1 from './ToDo1';
+
+const ToDo = () => {
+  const boxStyle = {
+    color: 'green',
+    border: '2px solid black',
+    width: '25%',
+    height: '5%',
+  };
+  const underline = {
+    textDecoration: 'underline',
+  };
+  const todoStyles = [
+    {
+      color: 'black',
+      backgroundColor: 'LightBlue',
+    },
+    {
+      color: 'red',
+      backgroundColor: 'LightBlue',
+    },
+    {
+      color: 'purple',
+      backgroundColor: 'LightBlue',
+    },
+  ];
+  const tasks = [
+    { activity: 'get out of bed', time: '06:00' },
+    { activity: 'Brush teeth', time: '06:15' },
+    { activity: 'Eat breakfast', time: '06:55' },
+  ];
+  return (
+    <div style={boxStyle}>
+      <h1 style={underline}>Todo List</h1>
+      {tasks
+        .map((task, index) => [task, todoStyles[index]])
+        .map((styledTask, index) => (
```

Comment:
Seems you're overcomplicating things a bit here.. 🙂 

1. Don't mix your `tasks` data with styling.
2. Add the styling for the `ToDo` component inside of it.

Don't you think that's simpler?

Apply the styçes